StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1120CommentsMustContainText.cs

```diff
@@ -66,12 +66,10 @@ private static void HandleSyntaxTree(SyntaxTreeAnalysisContext context)
 
         private static void HandleMultiLineComment(SyntaxTreeAnalysisContext context, SyntaxTrivia multiLineComment)
         {
-            var nodeText = multiLineComment.ToString();
+            var commentText = multiLineComment.ToString();
 
-            // We remove the /* and the */ and determine if the comment has any content.
-            var commentText = nodeText.Substring(2, nodeText.Length - 4);
-
-            if (string.IsNullOrWhiteSpace(commentText))
+            // We and determine if the comment has any content between /* and */
```

Comment:
:exclamation: The new wording doesn't make sense. Perhaps you meant to leave out the first 'and'?